packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -128,7 +127,6 @@ class AotAssemblyRelease extends AotAssemblyBase {
   List<Source> get inputs => const <Source>[
     Source.pattern('{FLUTTER_ROOT}/packages/flutter_tools/lib/src/build_system/targets/ios.dart'),
     Source.pattern('{BUILD_DIR}/app.dill'),
-    Source.pattern('{PROJECT_DIR}/.packages'),
```

Comment:
@jmagman this did. but gen_snapshot/aot doesn't even need the package config file, so removed

Reply:
Ah I see the extra commit on https://github.com/flutter/flutter/pull/66776 https://github.com/flutter/flutter/pull/67589/commits/5a817f5606b138c4e251b56fcd52fefe1e5d73ee